### ISSUE TITLE
Send email with attachments not working on Android 13

### ISSUE THREAD

**Issue by kostadin24** [sidlatau/flutter_email_sender#87]
This code works fine on Android versions before 13:
final Email email = Email(
      body: _body,
      subject: 'Network information',
      recipients: ['$_supportEmail'],
      //cc: ['cc@example.com'],
      //bcc: ['bcc@example.com'],
      attachmentPaths: attachments,//Files are in app document folder
      isHTML: false,
    );

On Android 13 it says: PlatformException(not_available, No email clients found!, null, null)
Version of plugin is: flutter_email_sender: ^5.1.0
I have mail client on my phone and it is selected as default.
----------
It works on Android 13 it if remove attachments.
Files are in app document folder. This folder was visible in previous androids. But with recent versions Android may be limit access to this folder.

**Comment by kostadin24** [comment]
EDIT: It works on Android 13 it if remove attachments:
attachmentPaths: null

**Comment by keitokusano** [comment]
I am also facing the same issue.

**Comment by explorer2050** [comment]
+1  same problem mentioned above seen on Pixel 5 "On Android 13 it throws: PlatformException(not_available, No email clients found!, null, null)". If user has only gmail installed, it will not work.  Are there any plans to fix this issue?

**Comment by rafaellop** [comment]
Please try adding this to your manifest:

```
            <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
            <queries>
                <intent>
                    <action android:name="android.intent.action.SENDTO" />
                    <data android:scheme="mailto" />
                </intent>
            </queries>
```

**Comment by kostadin24** [comment]
I have this added already. It helped me before plugin started targeting
android 13. But now is not the solution.

**Comment by Douglal** [comment]
Hello, I am having exactly the same issue as kostadin24
I have added the <intent> statement in Android manifest and flutter_email_sender works fine with no attachement. However with an attachement it fails with this error on Android 13: 
   PlatformException(not_available, No email clients found!, null, null)
flutter_email_sender is a great package (thanks sidlatau) and has worked reliably in my live app for 2 years on Android and iOS
**sidlatau, please fix this as soon as possible as my Path Checker app has hundreds of users!**

**Comment by Douglal** [comment]
The attachment in my app is a .jpg image selected from the gallery or the camera using image_picker. The picked image displays fine on screen using Image.file(File(pickedFile.path)) but flutter_email_sender now fails on Android 13 with attachments.add(pickedFile.path).
This error happens on Google Pixel 4a and 6a.
I hope that this extra information helps you to solve the problem quickly for us all.

**Comment by sidlatau** [comment]
Hmm, I am unable to reproduce the problem. Does an example app work for you? Does an error appear in an emulator? How many files you are attaching?

**Comment by Douglal** [comment]
Thank you for the quick reply. As you can see above several people are having this same problem. My Path Checker app is using your example code (using just one attachment) and it still works fine for iPhone and for earlier versions of Android (it worked correctly on my old Samsung Galaxy S9 but doesn't on my new Galaxy Pixel 6a). I am using Visual Studio Code on Windows 10 and the error "PlatformException(not_available, No email clients found!, null, null)" occurs in the Pixel 4a emulator (but the email is created with an attachment and no error in the Galaxy S9 emulator).
The pickedFile.path is like this: /data/user/0/com.colton.path_checker/cache/image_picker12345678901234567890.jpg
Do you think the way that Android 13 caches images from the gallery and from the camera is part of the problem?
I have just given you access to my app's source code repository at github.com/Douglal/path_checker

**Comment by kostadin24** [comment]
I'm attaching files stored in app document folder. They are binary data, not image/video/etc. so caching in gallery is not my case.

**Comment by mahesh-resideo** [comment]
For fix refer: https://pub.dev/packages/simple_email_sender

**Comment by sidlatau** [comment]
I was able to reproduce a problem. It should be fixed in [v5.2.0](https://pub.dev/packages/flutter_email_sender/versions/5.2.0). If you still have a problem, please reopen this or create a new issue.

**Comment by BarreseCA** [comment]
i have the same problem: PlatformException(not_available, No email clients found!, null, null) if i send e-mail with attachmentPaths in android 13.


final File? imageFile = image != null ? await _createImageTempDirectory(image, imageName) : null;

    final Email email = Email(
      subject: subject,
      recipients: recipients,
      cc: cc,
      body: body,
      attachmentPaths: imageFile != null ? [imageFile.path] : null,
      isHTML: false
    );
    
    
i read image from:
Future<File> _createImageTempDirectory(Uint8List image, String imageName) async {
    final Directory tempDir = await getTemporaryDirectory();
    final String filePath = '${tempDir.path}/$imageName.jpg';

    return await File(filePath).writeAsBytes(image);
  }
